modules/remote-state-s3-backend/variables.tf

```
@@ -0,0 +1,17 @@
+variable "s3_bucket_name" {
+  description = "If override_s3_bucket_name is true, use this bucket name for replica instead of dynamic name with bucket_prefix"
```

This doesn't make sense. Probably copy-paste from the original module.

```suggestion
  description = "Name of the bucket. Replica bucket name will be suffixed with `-replica`."
```